hooks/post-checkout

```diff
@@ -11,13 +11,17 @@ echo "--- Sending Hangouts Message"
 WEBHOOK_URL="${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_WEBHOOK_URL?}"
 CHAT_URL=${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_CHAT_ENDPOINT?}
 MESSAGE=$(echo "${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_MESSAGE?}" | envsubst)
-WEBHOOK_URL2="${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_WEBHOOK_URL2?}"
-WEBHOOK_KEY="${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_WEBHOOK_KEY?}"
-USERTOKEN_FAIL="${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_USERTOKEN_FAIL?}"
-USERTOKEN_PASS="${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_USERTOKEN_PASS?}"
 TYPE=${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_TYPE}
-CARD_DETAILS=${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_CARD_DETAILS}
-ROWS=$(echo "${BUILDKITE_PLUGIN_HANGOUTS_NOTIFY_ROWS?}" | envsubst)
+if [ $TYPE == 'CARD' ]
+  then
+  TYPE="KeyValueCARD"
```

Comment:
1. I don't think resetting type value here makes much sense as it means that plugin clients still have to pass `CARD` value for `TYPE` parameter ( given this `if [ $TYPE == 'CARD' ]`)
 
    -  I wanted us to keep backward compatibility by having a **separate** `if` statement that normalises TYPE to a supported value `KeyValueCARD` like that:
    ```bash
    if [ $TYPE == 'CARD' ]
      then
      TYPE="KeyValueCARD"
    fi
    ```

     - After TYPE was normalised to be `KeyValueCard` you can use it in `if` statement on line 15. 

    ```bash
    if [ $TYPE == 'KeyValueCARD' ]
      then
      # some code here
    fi
    ```
2. Could you update the documentation to say that `CARD` is redundant and won't be supported in future, plus `KeyValueCard `should be used instead?